src/components/Header/Header.js

```diff
@@ -1,11 +1,38 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import { NavLink, Link } from 'react-router-dom';
+import { ReactComponent as Logo } from '../../images/logo-black.svg';
+
+export function Header(props) {
+  const assignLogin = () => {
+    const { user } = props;
+    let loginButton;
+    if (user.length) {
+      loginButton = <NavLink to="/" className="nav-link">LOGOUT</NavLink>;
+    } else {
+      loginButton = <NavLink to="/login" className="nav-link">LOGIN</NavLink>;
+    }
+
+    return loginButton;
+  };
 
-function Header() {
   return (
-    <div>
-      <h1> HEADER HEADER HEADER HEADER HEADER</h1>
-    </div>
+    <header>
+      <Link to="/" className="logo-a-tag">
+        <Logo className="logo" />
+        <h1>Colorado</h1>
+      </Link>
+      <section>
+        <NavLink className="nav-link" to="/">FIND A DOJO</NavLink>
+        <NavLink className="nav-link" to="/register">REGISTER</NavLink>
+        {assignLogin()}
+      </section>
+    </header>
   );
 }
 
-export default Header;
+export const mapStateToProps = ({ user }) => ({
+  user
+});
+
+export default connect(mapStateToProps, null)(Header);
```

Comment:
for the connect, the null is not needed if u don't have mapDispatch? Unless the linter is complaining about it??

Reply:
I know that if MSTP is not being used that you should put null in the connect statement, not sure about MDTP. But the linter is not upset !